src/app/app.component.ts

```diff
@@ -224,11 +237,11 @@ export class AppComponent {
   /**
    * View a specific page from within the pages array.
    */
-  public viewPage(page: SearchPage) {
+  public viewPage(page: SearchPage): void {
     this.selectedPage = page;
     console.log(`Showing page ${page}`);
```

Comment:
Is this console.log necessary?

Reply:
Not any more - it was useful when trying to debug why the "last page" kept on being repeated.